### DISCUSSION TITLE
電卓作ろう

### DISCUSSION THREAD

**Discussion by satsukizzz** [satsukizzz/n-playspace-http#22] [Ideas]
皆さんお疲れ様です！
本日の授業でアイディアについて先生が触れてくださいました。

**_電卓！_**

ということで、早速Discussions作りますので気ままにやっていきますか 🤔

**Comment by satsukizzz** [comment]
最初の大枠としての電卓の形作りますね。
バグバグ&使いにくいなどあればみんなで直していきましょ！

**Reply by kenton116** [reply]
了解です！
